mypy.ini

```diff
@@ -0,0 +1,59 @@
+[mypy]
+ignore_missing_imports = False
+
+[mypy-_strptime]
+ignore_missing_imports = True
+
+[mypy-clickhouse_driver]
+ignore_missing_imports = True
+
+[mypy-confluent_kafka]
+ignore_missing_imports = True
+
+[mypy-confluent_kafka.admin]
+ignore_missing_imports = True
```

Comment:
Does this not get caught in the `confluent_kafka` import above?

Reply:
Not unless we'd do `mypy-confluent_kafka.*`

Reply:
Oh, okay. Thanks for clarifying.